src/io/keyboard.js

```diff
@@ -1,11 +1,16 @@
 var Cast = require('../util/cast');
 
-function Keyboard () {
+function Keyboard (rt) {
     /**
      * List of currently pressed keys.
      * @type{Array.<number>}
      */
     this._keysPressed = [];
+    /**
+     * For activate hat
+     * @type{Runtime}
+     */
+    this._rt = rt;
```

Comment:
Let's make the argument `runtime` and `this.runtime` to avoid introducing new names.